test/Microsoft.TestPlatform.CrossPlatEngine.UnitTests/DataCollection/InProcDataCollectorTests.cs

```diff
@@ -79,6 +79,27 @@ public void InProcDataCollectorShouldInitializeIfAssemblyContainsAnyInProcDataCo
             Assert.AreEqual(this.inProcDataCollector.AssemblyQualifiedName, typeInfo.AssemblyQualifiedName);
         }
 
+        [TestMethod]
+        public void InProcDataCollectorLoadCoverlet()
```

Comment:
this test verify that also if aqn shows version 1.0.0.0 we correctly load a version 9.9.9.9 of lib